src/Nethermind/Nethermind.Synchronization.Test/Mocks/FirstFree.cs

```diff
@@ -19,17 +19,30 @@ public static FirstFree Instance
         {
             get
             {
-                if (_instance is null) LazyInitializer.EnsureInitialized(ref _instance, () => new FirstFree());
+                if (_instance is null) LazyInitializer.EnsureInitialized(ref _instance, () => new FirstFree(false));
 
                 return _instance;
             }
         }
 
-        private FirstFree()
+
+        private static FirstFree? _replaceableInstance;
+        public static FirstFree ReplaceableInstance
+        {
+            get
+            {
+                if (_replaceableInstance is null) LazyInitializer.EnsureInitialized(ref _replaceableInstance, () => new FirstFree(true));
+
+                return _replaceableInstance;
+            }
+        }
```

Comment:
Fancy way of writing:

```suggestion
        public static FirstFree ReplaceableInstance => _replaceableInstance ??= new FirstFree(true);       
```